qa/smoke-test-http/src/test/java/org/elasticsearch/http/HttpCompressionIT.java

```diff
@@ -57,6 +64,30 @@ public void testUncompressedResponseByDefault() throws IOException {
         response = client().performRequest(request);
         assertEquals(201, response.getStatusLine().getStatusCode());
         assertNull(response.getHeader(HttpHeaders.CONTENT_ENCODING));
+        assertThat(response.getEntity(), is(not(instanceOf(GzipDecompressingEntity.class))));
+    }
+
+    public void testCompressesGetDocumentResponseIfRequested() throws IOException {
```

Comment:
shall we rename this to the existing test name `testCompressesResponseIfRequested` and delete the old version? (as this is more thorough as it's reading the body)

Reply:
Sure, just fixed it